SU-HSDO/suhumsci
# hotfix: Mobile mega menu doesn't open on non-active trail pages

# READY FOR REVIEW

## Summary
- Fix for mobile mega menu doesn't open on non-active trail pages
- Fix all mega menu trees are open by default instead of just the active trail

## Need Review By (Date)
asap

## Urgency
high

## Steps to Test
Review the current issue on production by attempting to toggle open the mobile menu on the homepage: https://music.stanford.edu 
1. Spin up `music.stanford.edu` with these code changes shrink your browser down to mobile sizes.
2. Verify you can now toggle open the mobile menu
3. Verify on the homepage that all menu items are closed by default because there is no active trail
4. Click on an internal page and open the mobile menu. Verify the active trail menu tree is open by default and the others are closed.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
